docs/language/run-time-types.md

```diff
@@ -31,7 +31,16 @@ Type<Int>() == Type<Int>()
 
 Type<Int>() != Type<String>()
 ```
+
+To get the run-time type's fully qualified type identifier, use the `let identifier: String` field:
+
+```cadence
+let type = Type<Int>()
+type.identifier  // is "Int"
+```
```

Comment:
Can you include what the fully qualified type identifier for a composite type looks like?

Reply:
Good idea! Will do

Reply:
Added in 2cf37fc